features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -0,0 +1,1274 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That file has been moved. If you move it too on your PR and commit, you should be down to a few lines of change instead of 1200 lines. The new location is without the "karaf" part.

Reply:
Got it. Thanks, corrected.